src/GovUk.Education.ExploreEducationStatistics.Common/Validators/ValidationUtils.cs

```diff
@@ -1,36 +1,43 @@
 ﻿#nullable enable
 using System;
+using System.Collections.Generic;
 using System.Linq;
-using System.Net;
-using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
-namespace GovUk.Education.ExploreEducationStatistics.Common.Validators
+namespace GovUk.Education.ExploreEducationStatistics.Common.Validators;
+
+public static class ValidationUtils
 {
-    public static class ValidationUtils
+    public static BadRequestObjectResult ValidationResult<TEnum>(params TEnum[] codes) where TEnum : Enum
     {
-        public static ActionResult ValidationResult(params Enum[] errors)
-        {
-            return ValidationResult(errors.Select(error => error.ToString()).ToArray());
-        }
-        
-        public static ActionResult ValidationResult(params string[] errors)
-        {
-            var problemDetails = new ValidationProblemDetails(
-                errors.ToDictionary(
```

Comment:
I think this might have been broken before? Adding the same empty key if you supplied multiple errors in the method parameters